Mixpanel/MixpanelAPI.cs

```diff
@@ -134,6 +134,20 @@ public static void Clear()
             EngageQueue.Clear();
         }
 
+        /// <summary>
+        /// Get value of super property. if such super property not created yet will return empty
+        /// </summary>
+        /// <param name="name"></param>
+        /// <returns></returns>
+        public static Value GetSuperProperty(string name)
```

Comment:
Okay. I understand. But such functionality exist in other Minpanel SDK implementation:
Android: [getSuperProperties()](http://mixpanel.github.io/mixpanel-android/index.html)
iOS Objective-C: [currentSuperProperties](https://mixpanel.github.io/mixpanel-iphone/Classes/Mixpanel.html#//api/name/currentSuperProperties)
iOS Swift: [currentSuperProperties](https://mixpanel.github.io/mixpanel-swift/Classes/MixpanelInstance.html#/s:8Mixpanel0A8InstanceC22currentSuperPropertiesSDySSypGyF)
I modify it a little bit. I add new method to get all super properties that return Value which collect all supper properties, and other one to get only one property that was requested. second method should have batter performance in case when we know what exactly property we are looking for.

Reply:
Ok I will let Mixpanel know and see what they think.